xml/System.IO/FileSystemAclExtensions.xml

```diff
@@ -56,6 +56,37 @@
 
 This extension method was added to .NET Core to bring the functionality that was provided by the `System.IO.DirectoryInfo.Create(System.Security.AccessControl.DirectorySecurity)` .NET Framework method.
```

Comment:
Unrelated to your change but a link here would probably be easier. 

```suggestion
This extension method was added to .NET Core to bring the functionality that was provided by the <xref:System.IO.DirectoryInfo.Create(System.Security.AccessControl.DirectorySecurity)> .NET Framework method.
```

Reply:
Good catch, thanks for changing it.